browser/js/actions/AuthActions.js

```diff
@@ -35,50 +44,29 @@ export function signup (credentials) {
   }
 }
 
-export function loginSuccess (user) {
-  return {
-    type: AUTH_LOGIN_SUCCESS,
-    id: user.id,
-    user
-  }
-}
-
 export function getLoggedInUser () {
   return (dispatch, getState)  => {
```

Comment:
Couldn't you simply export *this* function? The outer wrapping function seems irrelevant here.